### PR TITLE
Jpn-29 duplicate characters dropdown bug

### PR DESCRIPTION
**Pull request by antivirak**

### CODE REVIEW

File: game/person_ren.py
Comment (CaptainAgilek) [inline game/person_ren.py:1]: ¨je nejaky duvod udelat z toho .rpy _ren.py?

Comment (antivirak) [reply game/person_ren.py:1]: líp pak funguje linter, když tam máme jen python kód

File: game/script.rpy [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 # The game starts here.
 label start:
+    $ persistent._clear()
```
Comment (antivirak) [inline game/script.rpy:5]: This probably wipes the Person instance attributes of older saved game when creating a new game. Need to think about this more.
When defining the instance using default keyword, it is treated as "persistent" variable - across all saves. When defining it using define keyword, it is saved when saving game if and only if it changed. The change of any attribute does not count as change of the instance itself.
We need something in between.

Comment (antivirak) [reply game/script.rpy]: Should be resolved

**Pull request merged**